test/val/val_cfg_test.cpp

```diff
@@ -1423,6 +1423,33 @@ TEST_F(ValidateCFG, OpReturnInNonVoidFunc) {
           "OpReturn can only be called from a function with void return type"));
 }
 
```

Comment:
Please add a test case which branches into a loop construct (bypassing the header).

Reply:
I'm not sure this can be done with the implementation of the validator. When restricted to reachable blocks every case I try runs afoul of backedge rules. Branching around the header is no good because it prevents blocks from being part of the loop construct. Branching back into the loop is an extra backedge. Continuing into the middle of the loop construct is caught already.

Reply:
Ok.  Sounds good to me.